File: lista3/zadanie1/Prostokat.java
class Prostokat extends Czworokat  {

    public Prostokat(int bok1, int bok2, int bok3, int bok4, int kat){
        super(bok1, bok2, bok3, bok4, kat);
        
    }

    public double obliczPole() {
        if (dajBok1() != dajBok2())
            return dajBok1() * dajBok2();

        else 
            return dajBok1() * dajBok2();
    }

    public void dajNazwe(){
        System.out.println("Jestem prostokatem");
    }
    
}

File: lista3/zadanie2/Prostokat.cpp
#include "Figura.hpp"
#include "Czworokat.hpp"
#include "Prostokat.hpp"

double Prostokat::obliczPole(){
    return dajBok1() * dajBok3();
}

void Prostokat::dajNazwe(){
    std::cout << "Jestem prostokatem" << std::endl;
}


